internal/policy/policy.go

```diff
@@ -765,3 +770,54 @@ func (s *State) findDelegationEntry(roleName string) (*tuf.Delegation, error) {
 		}
 	}
 }
+
+func ListRules(ctx context.Context, repo *git.Repository) ([]*DelegationDepth, error) {
```

Comment:
missing docstring

Reply:
Also, please move this function above so all the private functions are at the end of the file.

Reply:
I have moved it and added a docstring